src/tokens/renec.tokenlist.json

```diff
@@ -144,6 +144,19 @@
       "extensions": {
       }
     },
+    {
+      "chainId": 102,
+      "address": "DA22tiueni7Sm8eha7vkWPsEacpJWSPYr2HQfeh9upUp",
+      "symbol": "VNLA",
+      "name": "VanillaSwap Token",
+      "decimals": 9,
+      "logoURI": "https://github.com/marv747/VanillaSwap/blob/e7e31396720c7354e831dbcc02a86860609423a7/icon-transparent5e1f.png",
```

Comment:
Pls upload this image to `/assets/mainnet/DA22tiueni7Sm8eha7vkWPsEacpJWSPYr2HQfeh9upUp` of this repo

Reply:
Pls I can't add a file because I don't have push rights I'm sending the photo to you so you can please help me upload it. Thanks 
![logo.png](https://user-images.githubusercontent.com/85713038/220104093-75e3cef1-b676-47cc-b529-d6eeaa79dd16.png)

Reply:
@marv747 You can add this image to this PR also

Reply:
Just copy that image to your local repo then commit and push